Project.toml

```diff
@@ -12,11 +12,13 @@ Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 Flatten = "4c728ea3-d9ee-5c9a-9642-b6f7d7dc04fa"
 FreezeCurves = "71e4ad71-e4f2-45a3-aa0a-91ffaa9676be"
+Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 KernelAbstractions = "63c18a36-062a-441e-b654-da1e3ab1ce7c"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 ModelParameters = "4744a3fa-6c31-4707-899e-a3298e4618ad"
 NCDatasets = "85f8d34a-cbdd-5861-8df4-14fed0d494ab"
+NamedTupleTools = "d9ec5142-1e00-5aa0-9d6a-321866360f50"
```

Comment:
The package is not very well maintentained and has some type piracy issues, if you use only one function from it, it might be worth it to just reimplement that function.